examples/chat.rs

```diff
@@ -157,13 +166,13 @@ fn main() {
 
         loop {
             match swarm.poll().expect("Error while polling swarm") {
-                Async::Ready(Some(_)) => {
-
-                },
+                Async::Ready(Some(_)) => {},
```

Comment:
Not sure if this should be changed; shouldn't there be a break and clean exit here? I am not quite sure about the return of [`Swarm::poll`](https://docs.rs/libp2p/0.13.2/libp2p/swarm/trait.NetworkBehaviour.html#tymethod.poll) or the custom `#[behaviour(poll_method = "...")]` poll method, which I why the sentence I added was left unfinished :)